blockchain/genesis/genesis.go

```diff
@@ -56,14 +56,15 @@ func defaultConfig() Genesis {
 			EnableGravityChainVoting: true,
 		},
 		Rewarding: Rewarding{
-			InitBalanceStr:                 unit.ConvertIotxToRau(1200000000).String(),
+			InitBalanceStr:                 unit.ConvertIotxToRau(200000000).String(),
 			BlockRewardStr:                 unit.ConvertIotxToRau(16).String(),
-			EpochRewardStr:                 unit.ConvertIotxToRau(300000).String(),
+			EpochRewardStr:                 unit.ConvertIotxToRau(12500).String(),
+			AleutianEpochRewardStr:         unit.ConvertIotxToRau(15000).String(),
 			NumDelegatesForEpochReward:     100,
 			ExemptAddrStrsFromEpochReward:  []string{},
-			FoundationBonusStr:             unit.ConvertIotxToRau(2880).String(),
+			FoundationBonusStr:             unit.ConvertIotxToRau(80).String(),
 			NumDelegatesForFoundationBonus: 36,
-			FoundationBonusLastEpoch:       365,
+			FoundationBonusLastEpoch:       8760,
```

Comment:
these change means we are changing epoch to 1 hour?

Reply:
No, just make it equivalent to 1yr if epoch length is 1hr, make the default consistent with the mainnet setting